packages/@lwc/template-compiler/package.json

```diff
@@ -19,14 +19,15 @@
         "build": "tsc"
     },
     "devDependencies": {
-        "@types/estree": "0.0.48",
+        "@types/estree": "0.0.50",
         "@types/he": "^1.1.1",
         "@types/parse5": "^6.0.1"
     },
+    "//": "Currently can't upgrade estree-walker to v3 because it breaks karma tests due to ESM: https://git.io/JuCcq",
```

Comment:
In order to upgrade estree-walker to v3 we would need to get rid of our CommonJS build output for `@lwc/template-compiler`. https://github.com/Rich-Harris/estree-walker/issues/26